addons/l10n_si/data/account.chart.template.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<openerp>
+    <data noupdate="1">
+        <record id="account_chart_si_gd" model="account.chart.template">
+            <field name="name">Kontni načrt za gospodarske družbe</field>
+            <field name="account_root_id" ref="account_account_"/>
+            <field name="tax_code_root_id" ref="account_tax_code_1"/>
+            <field name="bank_account_view_id" ref="account_account_110000"/>
```

Comment:
<field name="bank_account_view_id" ref="account_account_110"/>

It's supposed to set the view type account.